extension/src/plots/paths/collect.ts

```diff
@@ -167,15 +170,17 @@ const collectOrderedPath = (
   const isPathLeaf = idx === pathArray.length
   const isMultiImgPlot = isMultiImgDir && isPathLeaf
 
-  if (acc.some(({ path: existingPath }) => existingPath === path)) {
-    return updateExistingPlotPath(
+  const existingPathInd = acc.findIndex(existing => existing.path === path)
+  if (existingPathInd >= 0) {
```

Comment:
[Q] would `!== -1` be easier to grok at a glance (I had to look up what `findIndex` returned for a miss).